mastering-plone/add-ons.rst

```diff
@@ -17,6 +17,11 @@ Some notable add-ons
 `collective.easyform <https://pypi.python.org/pypi/collective.easyform>`_
   A form generator and the successor to `Products.PloneFormGen <https://docs.plone.org/develop/plone/forms/ploneformgen.html>`_
 
+.. image:: _static/add_ons_easyform_1.png
```

Comment:
Indent this section same as the text to make it more obvious that it belongs to the easyform addon. 

If you add a `:scale: 50%` the image will be additionally rendered as a link to the full size. Please always use a scale!

You should always use `.. figure::` instead of `image`. `figure` allows you to add a caption and legend to the image. See http://docutils.sourceforge.net/docs/ref/rst/directives.html#figure. In our case we at least sometimes want to add a caption.

Reply:
The caption here could be: "A simple form created with collective.easyform"